package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon -e scss --exec npm run scss",
-    "scss": "node-sass public/assets/scss/app.scss public/assets/css/app.css --output-style=compressed"
+    "scss": "node-sass public/assets/scss/app.scss public/assets/css/app.css --output-style=compressed",
+    "checkout-pb": "git checkout ft-UI-for-form-171431848",
```

Comment:
Can we remove you git command you added to the NPM scripts? They seem redundant and not valuable to other branches apart from this.